sinonjs/referee-sinon
# New mochify

#### Purpose (TL;DR) - mandatory

I'm working on a rewrite of mochify which aims to solve #114, is more modular and works with any bundler.

#### Background (Problem in detail)  - optional

Mochify development happens on the `rewrite` branch here: https://github.com/mantoni/mochify.js/tree/rewrite

#### Solution  - optional

We're using esbuild to bundle the browser package, but mochify depends on browserify. With the new version of mochify any bundler can be used.

The new `webdriver` plugin uses the official `webdriver` module. While the early stage seems to work fine with Safari, I'm seeing different issues with all other browsers.

#### How to verify - mandatory

1. Check out this branch
2. `npm install`
3. `npm run test-headless`
4. `npm run test-safari`
5. `npm run test-firefox` (currently fails for me)
6. `npm run test-ie` (currently times out for me)

#### Checklist for author
